build.gradle

```diff
@@ -17,29 +21,13 @@ buildscript {
     }
 }
 
-apply plugin: 'groovy'
-apply plugin: 'spring-boot'
-apply plugin: 'maven-publish'
-apply plugin: 'com.ofg.uptodate'
-apply plugin: 'docker'
-apply plugin: 'accurest'
-apply plugin: 'com.blogspot.toomuchcoding.testprofiler'
-
 ext {
     versionPrefix = "CD"
-    buildNrLoc = project.hasProperty('buildNr') ? "${buildNr}" : "000"
-    currentVersion = "${versionPrefix}-${buildNrLoc}"
-    projectGroupId = 'com.ofg'
-
-    contractsDir = file("${project.rootDir}/repository/mappings/com/ofg/twitter-places-analyzer")
-    wiremockStubsOutputDirRoot = file("${project.buildDir}/production/${project.name}-stubs/")
-    wiremockStubsOutputDir = new File(wiremockStubsOutputDirRoot, 'mappings/')
-
-    //microInfraSpring and versions of other dependencies were moved to gradle.properties
+    buildNrLoc = project.hasProperty('buildNr') ? "${buildNr}" : "CD-000"

```

Comment:
Let's just leave it for now - eventually we can go with Axion but let's try to be consitent with the inconsistencies that we have in our Nexus ;) WDYT?